sdk/src/constants/perpMarkets.ts

```diff
@@ -596,6 +709,8 @@ export const MainnetPerpMarkets: PerpMarketConfig[] = [
 		oracle: new PublicKey('Gcm39uDrFyRCZko4hdrKMTBQsboPJHEd4RwnWhWFKr9a'),
 		launchTs: 1718021389000,
 		oracleSource: OracleSource.SWITCHBOARD,
+		pythFeedId:
+			'0x82595d1509b770fa52681e260af4dda9752b87316d7c048535d8ead3fa856eb1',
```

Comment:
looks like some markets missing feedIds? they were probably added after this PR was first made